Sources/Sentry/SentryClient.m

```diff
@@ -585,6 +584,30 @@ - (void)captureUserFeedback:(SentryUserFeedback *)userFeedback
     [self.transportAdapter sendUserFeedback:userFeedback];
 }
 
+- (void)captureFeedback:(SentryFeedback *)feedback withScope:(SentryScope *)scope
+{
+    SentryEvent *feedbackEvent = [[SentryEvent alloc] init];
+    feedbackEvent.eventId = feedback.eventId;
+    feedbackEvent.type = SentryEnvelopeItemTypeFeedback;
+
+    NSDictionary *serializedFeedback = [feedback serialize];
+
+    NSMutableDictionary *context = [NSMutableDictionary dictionary];
+    context[@"feedback"] = serializedFeedback;
+    feedbackEvent.context = context;
```

Comment:
`l`: A bit shorter and requires memory if I'm not mistaken.
```suggestion
    feedbackEvent.context = @{ @"feedback": serializedFeedback };
```

Reply:
This will still have to allocate memory, but, I can use `-[NSMutableDictionary dictionaryWithCapacity:]` to prevent reallocation and moving. I also as a rule will never use dictionary literals in production implementation again due to past experience with crashes (tests are fine).